src/main/kotlin/com/carneirofatima/vehicleterminal/ValidationRequest.kt

```diff
@@ -0,0 +1,26 @@
+package com.carneirofatima.vehicleterminal
+
+import com.carneirofatima.vehicleterminal.dao.DriverDao
+import com.carneirofatima.vehicleterminal.to.DriverTO
+import org.springframework.stereotype.Component
+
+@Component
+class ValidationRequest(
+    private val dao: DriverDao
+) {
+
+    fun validateEdit(form: DriverTO) {
+        if (form.driverId == 0) {
```

Comment:
This validation you can put on DriverTO class with annotations, using `@field:Min` or `@field:Positive`